nikola/data/themes/base/templates/tag.tmpl

```diff
@@ -43,7 +43,8 @@
     %if posts:
     <ul class="postlist">
     % for post in posts:
-        <li><a href="${post.permalink()}" class="listtitle">${post.title()|h}</a> <time class="listdate" datetime="${post.formatted_date('webiso')}" title="${post.formatted_date(date_format)}">${post.formatted_date(date_format)}</time></li>
+        <li>
```

Comment:
I improved the item visualization in the base tamplate. It was show as:
Title Date --- (dash)
The dash is added by the date style. I changed it to be
Date --- Title
as in other templates I saw. I used the same pattern in author.tmpl. Maybe there is an extra new line.